frontend/cypress/integration/featureFiles/kiali_cookie.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Kiali login cookie
+
+  Smoke to verify cy.login function is working and stores cookies into session
+  
+
+  Background: 
+    Given user is at administrator perspective
+  
+  Scenario: Open Kaili home page
+    And user opens base url
+    Then user see console in URL
+
+  Scenario: Open Kaili home page2
+    And user opens base url
+    Then user see console in URL
+
+  
```

Comment:
I don't think we'll really need this feature file in the future since most other tests will use the `Given user is at administrator perspective` line thereby testing the cookie usage and cypress setting a cookie is not really a user facing feature we want to test. It can be removed later though as tests are added/updated to use the `Given ...` line.

Reply:
Lets leave it for now, It's good smoke, possibly I will rewrite it a little bit to assert cookie have all parameters it should have